metrics-extensions/modules/grafana-dashboards/dashboards/ballerina-metrics-grafana-dashboard-prometheus.json

```diff
@@ -2281,17 +2545,27 @@
     "list": [
       {
         "allValue": null,
-        "current": {},
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
         "datasource": "${DS_PROMETHEUSDS}",
+        "definition": "label_values(requests_total_value,instance)",
         "hide": 0,
         "includeAll": true,
         "label": "instance",
         "multi": true,
         "name": "instance",
         "options": [],
-        "query": "label_values(instance)",
-        "refresh": 1,
+        "query": "label_values(requests_total_value,instance)",
+        "refresh": 2,
```

Comment:
Do we need the change in `refresh` ? Shall we revert this change in all the places ?